<launch>
  <arg name="bag" />
  <arg name="out" default="frame%04d.jpg" />
  <node pkg="rosbag" type="play" name="rosbag" args=" --clock $(arg bag)"/>
  <node name="extract" pkg="image_view" type="extract_images" respawn="false" output="screen" cwd="ROS_HOME">
    <param name="filename_format" value="$(arg out)" />
    <remap from="image" to="/camera/rgb/image_rect_color"/>
  </node>
</launch>
